libr/core/disasm.c

```diff
@@ -1009,6 +1009,10 @@ static void ds_build_op_str(RDisasmState *ds, bool print_color) {
 	core->parser->relsub = r_config_get_i (core->config, "asm.relsub");
 	core->parser->regsub = r_config_get_i (core->config, "asm.regsub");
 	core->parser->relsub_addr = 0;
+	if (core->parser->relsub && ds->analop.type == R_ANAL_OP_TYPE_LEA
+	    && (ut64)ds->analop.ptr == ds->vat + ds->analop.size + ds->analop.disp) {  // PC-relative
```

Comment:
imho the op->ptr should be pointing to the right address without having to do this pc relative addressing in disasm.c, the anal backend should give the right pointer. and disp shouldnt be used at all

Reply:
ok 2a5f7c8b1a50e550e751d2b3f18a9b15a74cbcba